kie-remote/kie-remote-jaxb-gen/pom.xml

```diff
@@ -292,11 +268,10 @@
               <goal>schemagen</goal>
             </goals>
             <configuration>
-              <includes>
-                <include>org/drools/**/*.java</include>
-                <include>org/jbpm/**/*.java</include>
-              </includes>
-              <verbose>true</verbose>
+              <sources>
+                <source>${project.basedir}/src/main/java/org/drools</source>
+                <source>${project.basedir}/src/main/java/org/jbpm</source>
+              </sources>
```

Comment:
The plugin docs (http://www.mojohaus.org/jaxb2-maven-plugin/Documentation/v2.2/schemagen-mojo.html#sources) says that `sources` is: "Parameter holding List of paths to files and/or directories which should be recursively searched for Java source files".